scripts/app.js

```diff
@@ -0,0 +1,93 @@
+const StagingUtils = require('./stagingUtils');
+
+async function main() {
+  const patchFlag = process.argv[2];
```

Comment:
When I ran make stage, I got a response that none of the fields were in .snootyenv, even though the DB_NAME was there. I had the thought that we could simplify this .env file to just accepting the secret. The rest can really be hardcoded. Can we just manage the secret from this file?